docs/DeveloperGuide.md

```diff
@@ -1496,16 +1503,22 @@ a “Timing” attribute to tutorials, enabling administrators to view and manag
 5. **Support for Education Levels in Tutorials**: Tutorials currently do not distinguish between education levels. To
 improve categorization and management, we plan to add an “Education Level” (eg, Secondary 1-Math, Junior College
 2-Chemistry), allowing administrators to group tutorials by levels such as primary, secondary, or advanced.
-6. **Improve UI for Screen Resizing**: The current UI can experience errors when resizing the screen. We plan to enhance
-UI responsiveness, particularly in handling smaller or larger window sizes, to improve usability across various screen configurations.
-7. **Batch Enroll/Unenroll for Multiple Students**: The current system allows enrolling or unenrolling one student at a
-time. We plan to add functionality for batch enrollments and unenrollments, allowing administrators to enroll or unenroll
-multiple students from a tutorial at once.
-8. **Batch Enroll/Unenroll for Multiple Tutorials**: Currently, students can only be enrolled or unenrolled from one
-tutorial at a time. We plan to add functionality to support enrolling or unenrolling a student from multiple tutorials
-simultaneously, improving efficiency.
-9. **Prevent Duplicate Attendance Records**: Currently, attendance can be added multiple times even if it’s within the
-same week for the same tutorial participation. We plan to add a check to prevent duplicate attendance within the same week, ensuring accurate attendance tracking.
+6. **Support For Special Characters in Names**: Currently, special characters such as '/' are not allowed in the name
+field.
+In the future, we plan to enhance the application to allow such characters in name inputs.
+7. **Batch Enroll/Unenroll for Multiple Students and Tutorials**: The current system allows enrolling or unenrolling one
+student into one tutorial at a time. We plan to add functionality for batch enrollments and unenrollments, allowing
+administrators to enroll or unenroll multiple students from a tutorial at once. As well as enroll and unenroll
+a student from multiple tutorials.
+8. **Enhance Error Message specificity**: Currently, certain error messages related to invalid user input are either too
+vague or not displayed at all. Specifically, if a user includes a prefix that does not fall within the command's scope,
+the command will not return an error message. Instead, it might process the prefix as a string input and passes it to
+other fields, potentially causing additional errors. Moving forward, we plan to restrict the use of invalid prefixes based on
+the specific command to ensure more precise error handling.
+9. **Allowing Multiple Attendance Records**: Currently, attendance can only be recorded once per tutorial per week.
+This is to ensure user do not accidentally key in the wrong attendance. We plan to allow multiple attendance within the
```

Comment:
can add "which results in duplicate weekly attendance"?

Reply:
ok will add to this line 'This is to ensure user do not accidentally key in the wrong attendance, which results in duplicate weekly attendance'